src/pramod/FrequencyOfGivenCharacterAssignment17.java

```diff
@@ -0,0 +1,25 @@
+package pramod;
+
+public class FrequencyOfGivenCharacterAssignment17 {
+	
+	void frequencyOfGivenCharacterNumber(String s)
+	{
+		int count=0;
+		for(int i=0;i<s.length();i++)
+		{
+			if(s.charAt(i)=='t')
```

Comment:
This code will only work for 't'. Instead of this make method parameterized and make your code generic. So it work for all letters